orca-pipelinetemplate/src/main/java/com/netflix/spinnaker/orca/pipelinetemplate/v1schema/graph/transform/ConfigStageInjectionTransform.java

```diff
@@ -117,11 +119,22 @@ private static void dfs(String stageId,
 
     state.put(stageId, Status.VISITING);
     StageDefinition stage = graph.get(stageId);
+    if (stage == null) {
```

Comment:
You know what ? I think that's me being lazy on rebuilding the graph after it has been mutated. I think I can get rid of that null check and handle rebuilding the graph. Keeps the dfs method clean that way